Why is plugin-import is peerDep of node-resolver?

I am tracking down some very strange shrinkwrap issues caused by eslint/eslint-plugin-import and I came across this:

https://github.com/benmosher/eslint-plugin-import/blob/71afde73b8d18a487671d1d4ae0534c0ca52e938/resolvers/node/package.json#L31

Why is eslint-plugin-import (hereafter "import") listed as a peerDep of eslint-import-resolver-node (hereafter "resolver")? Is my understanding correct that in no scenario does resolver expect to be a sibling of import?

Resolver is already a direct dependency of import. And resolver does not `require` or rely on any of the functionality of import to function.

I've been able to boil this down to a minimal reproducible issue, that is actually causing production problems for us: (only manifests on npm2)

`npm init -f; npm install --save eslint-plugin-import@1.8.0 eslint@^2.9.0; npm shrinkwrap; rm -r node_modules; npm install; npm shrinkwrap; rm -r node_modules; npm install; npm shrinkwrap; rm -r node_modules; npm install;`

The above demonstrates a scratch package with direct deps installed: `eslint ^2.9.0` and `eslint-plugin-import@1.8.0`

Given such a package, running the following commands: `npm shrinkwrap; rm -r node_modules; npm install` will fail on the third install due to versioning conflicts of eslint:

```
npm ERR! install trying to install 3.0.1 to /private/tmp/take2/node_modules/eslint
npm ERR! install but already installed versions [ '2.13.1' ]
... [100s of lines snipped]
```

--- a/resolvers/node/package.json
+++ b/resolvers/node/package.json
@@ -27,9 +27,6 @@
     "object-assign": "^4.0.1",
     "resolve": "^1.1.6"
   },
-  "peerDependencies": {
-    "eslint-plugin-import": ">=1.4.0"
-  },
   "devDependencies": {
     "chai": "^3.4.1",
     "mocha": "^2.3.4"

